File: src/HelloTizenFrame.cpp
#include "HelloTizenFrame.h"
#include "HelloTizenFormFactory.h"
#include "HelloTizenPanelFactory.h"
#include "AppResourceId.h"

using namespace Tizen::Base;
using namespace Tizen::Ui;
using namespace Tizen::Ui::Controls;
using namespace Tizen::Ui::Scenes;

HelloTizenFrame::HelloTizenFrame(void)
{
}

HelloTizenFrame::~HelloTizenFrame(void)
{
}

result
HelloTizenFrame::OnInitializing(void)
{
	// Prepare Scene management.
	SceneManager* pSceneManager = SceneManager::GetInstance();
	static HelloTizenFormFactory formFactory;
	static HelloTizenPanelFactory panelFactory;
	pSceneManager->RegisterFormFactory(formFactory);
	pSceneManager->RegisterPanelFactory(panelFactory);
	pSceneManager->RegisterScene(L"workflow");

	// Go to the scene.
	result r = E_FAILURE;
	r = pSceneManager->GoForward(SceneTransitionId(ID_SCNT_MAINSCENE));

	// TODO: Add your initialization code here
	return r;
}

result
HelloTizenFrame::OnTerminating(void)
{
	result r = E_SUCCESS;

	// TODO:
	// Add your termination code here
	return r;
}

File: HelloTizen/Source/HelloTizenFrame.cs

using Tizen;
using Tizen.Base;
using Tizen.Ui.Scenes;


class HelloTizenFrame : Tizen.Ui.Controls.Frame
{
	public void SetName(string name)
	{
	}

	ulong OnInitializing()
	{
		// Prepare Scene management.
		SceneManager sceneManager = SceneManager.GetInstance();
		sceneManager.RegisterFormFactory(new HelloTizenFormFactory());
		sceneManager.RegisterPanelFactory(new HelloTizenPanelFactory());
		sceneManager.RegisterScene(new Tizen.Base.String("workflow"));

		// Go to the scene.
		ulong r = Tizen.Constants.E_FAILURE;
		r = sceneManager.GoForward(new Tizen.Base.String(Constants.ID_SCNT_MAINSCENE));

		// TODO: Add your initialization code here
		return r;
	}

	ulong OnTerminating()
	{
		ulong r = Tizen.Constants.E_SUCCESS;

		// TODO:
		// Add your termination code here
		return r;
	}

}
